test/regressions/index.test.js

```diff
@@ -184,7 +184,7 @@ async function main() {
         printButton.click();
       });
 
-      await sleep(2000);
+      await sleep(6000);
```

Comment:
Looks like we need to increase test timeout as well

Reply:
Weird, the timeout is already at 10s. Let's try another value first.

```suggestion
      await sleep(5000);
```